<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>0</y>
      <width>701</width>
      <height>600</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="username">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>200</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>360</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>USERNAME:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>240</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>240</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>PASSWORD:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>300</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>251</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Papyrus</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Memory Trainer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>500</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>330</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Game1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>330</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Game2Easy</string>
      </property>
     </widget>
     <widget class="QLabel" name="welcome">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>421</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="scorelabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="profile">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>120</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>370</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Game2Normal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>410</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Game2Hard</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>126</x>
        <y>233</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="welcome1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>265</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Game1 Highscore:</string>
      </property>
     </widget>
     <widget class="QLabel" name="nname">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>233</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="uusername">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="game1score">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>303</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>340</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Game2 Highscore:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backtom">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>430</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="game2score">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>340</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5"/>
    <widget class="QWidget" name="page_6"/>
    <widget class="QWidget" name="page_7"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>username</tabstop>
  <tabstop>password</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>profile</tabstop>
  <tabstop>backtom</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_5</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
